scripts/node_status.sh

```diff
@@ -0,0 +1,27 @@
+#/bin/sh
+
+NODES=$1
+if [ -z $NODES ]
+then
+    NODES=1
+fi
+
+echo "**** Number of nodes to be check for the status: $NODES ****"
+
+IP="$(dig +short myip.opendns.com @resolver1.opendns.com)"
```

Comment:
If for some reason this command does not work we should take `127.0.0.1` as default